### PR TITLE
Test packaged EOSIO Linux artifacts against standard dockers.

### PR DESCRIPTION
**Pull request by williamblevins**
## Change Description

Updates for CICD to test packaged EOSIO Linux artifacts against standard dockers: `*.deb` and `*.rpm` files.

## Change Type
**Select *ONE*:**
- [ ] Documentation

- [ ] Stability bug fix

- [x] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [x] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Review by kj4ezj — APPROVED** [review]
💽

File: .cicd/test-package.run.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+echo '+++ :minidisc: Installing EOSIO'
+
+if [[ $(apt-get --version 2>/dev/null) ]]; then # debian family
```
Comment (jgiszczak) [inline .cicd/test-package.run.sh:6]: Is this somehow better than simply performing `which apt-get`?  `which` returns success only if the target exists and is executable.

**Review by jgiszczak — APPROVED** [review]
Approved, with a question.

File: .cicd/test-package.run.sh (hunk 1 above)
Comment (williamblevins) [reply .cicd/test-package.run.sh:6]: It's functionally equivalent, this way may be slightly more verbose is the only thing.

**Pull request merged**